docs/content/preview/releases/yba-releases/v2.20.md

```diff
@@ -55,6 +55,114 @@ For more information, see [#20827](https://github.com/yugabyte/yugabyte-db/issue
 
 {{< /warning >}}
 
+## v2.20.3.0 - April 9, 2024 {#v2.20.3.0}
+
+**Build:** `2.20.3.0-b68`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.20.3.0/yugabytedb-2.20.3.0-b68-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.20.3.0/yugabytedb-anywhere-2.20.3.0-b68-third-party-licenses.html)
+
+### Download
+
+<ul class="nav yb-pills">
+ <li>
+   <a href="https://downloads.yugabyte.com/releases/2.20.3.0/yba_installer_full-2.20.3.0-b68-linux-x86_64.tar.gz">
+     <i class="fa-brands fa-linux"></i>
+     <span>Linux x86</span>
+   </a>
+ </li>
+</ul>
+
+### New features
+
+* [Migrate from Replicated to YBA Installer](../../../yugabyte-platform/install-yugabyte-platform/install-software/installer/#migrate-from-replicated). Ability (for current customers who previously installed YBA using Replicated) to migrate to the improved YBA Installer for upcoming upgrades. The new installer offers enhanced speed, simplicity, and robustness. Note that Replicated will reach its end-of-life by the conclusion of 2024. Users are recommended to migrate to the YBA Installer before this deadline for a smoother transition. PLAT-8364
+* [Task Retryability](../../../yugabyte-platform/manage-deployments/retry-failed-task/) for VM-based universes. You can now address the cause of a failure for a complex operation and retry. For example, if adding nodes to a universe fails due to a cloud quota limit, you can fix the quota issue and retry the task. PLAT-6141
+* [Delphix integration](../../../integrations/delphix/). Delphix now officially supports YugabyteDB for its database virtualization and data masking solutions.
+* [Commvault integration](../../../integrations/commvault/). Commvault officially supports backup and restore of YugabyteDB.
+* [Automated OS Patching and upgrades for public cloud](/stable/yugabyte-platform/manage-deployments/upgrade-nodes-csp/). With just a few clicks, you can non-disruptively upgrade your Linux OSs to the latest security patch levels, or even to newer or different Linux OSs. PLAT-9012
+* Asymmetric Network proxy settings. For a given universe, network proxy settings can now be specified at per region granularity (rather than per universe granularity). For example, beneficial for multi-cloud universes sending their backups to a single cloud bucket. {{<badge/ea>}} PLAT-12604
```

Comment:
```suggestion
* Asymmetric Network proxy settings. For a given universe, network proxy settings can now be specified at per region granularity (rather than per universe granularity). This is beneficial in multi-cloud universes to, for example, send backups to a single cloud bucket. {{<badge/ea>}} PLAT-12604
```